terraform/shared/shared.tf.json.template.py

```diff
@@ -539,6 +539,7 @@ def conformance_pack(name: str) -> str:
                 a.name: {
                     'alarm_name': config.qualified_resource_name(a.name, suffix='.alarm'),
                     'comparison_operator': 'GreaterThanThreshold',
+                    'datapoints_to_alarm': 1,
                     'evaluation_periods': 1,
```

Comment:
```suggestion
                    'evaluation_periods': 1,
                    'datapoints_to_alarm': 1,
```

For consistency, but also logically.